t/146-timer-every.t

```diff
@@ -0,0 +1,381 @@
+# vim:set ft= ts=4 sw=4 et fdm=marker:
```

Comment:
Please rename this test file to `t/152-timer-every.t` since we already have a `t/146-*.t` file.